src/platform/unittest/TlsTest.cpp

```diff
@@ -820,6 +866,47 @@ TEST_F(TlsTest, CertificateError)
     }
 }
 
+#ifdef _WIN32
+TEST_F(TlsTest, DeferredCertificateValidationAllow)
```

Comment:
Does the AZP YAML need to be updated to run this test on Schannel builds?

Reply:
Nope. But related to this, I'm going to slightly modify how this work, and make this test always run, but just exit silently if the sec config couldn't be created (i.e. unsupported by the platform).